### DISCUSSION TITLE
Where are language aliases defined?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [nvim-treesitter/nvim-treesitter#3220] [Q&A]
I opened this issue on the markdown syntax: https://github.com/ikatyang/tree-sitter-markdown/issues/49

But, to summarize here: hbs is a subset of glimmer, is there a way to define `hbs` as an alias?

**Comment by theHamsta** [comment]
At the moment there is no way to define an alias. This is something thing that I suggested earlier and needs to be implemented in upstream Neovim.

See https://github.com/neovim/neovim/pull/18317b or https://github.com/neovim/neovim/pull/17127. 

There is a file type to language mapping in parser.lua but I think it is not used for injections
